src/theme/DocSidebarItem/Category/index.js

```diff
@@ -9,7 +9,7 @@ import {
 } from "@docusaurus/theme-common";
 import {
   isActiveSidebarItem,
-  findFirstCategoryLink,
```

Comment:
This function is no longer being exported. I think the replacement covers the functionality of the old function.